### ISSUE TITLE
Error running git upon startup

### ISSUE THREAD

**Issue by xt0rted** [desktop/desktop#2984]
### Description

For the last version or two I receive an error immediately upon opening the application. I never tried changing my settings, it just started after it updated (I think it skipped a handful of releases which might be the reason as this doesn't happen on my other system).

```
Error running git [config --global filter.lfs.clean git-lfs clean -- %f]: 'warning: filter.lfs.clean has multiple values
error: cannot overwrite multiple values with a single value
       Use a regexp, --add or --replace-all to change filter.lfs.clean.' 'exit status 5'
Run `git lfs install --force` to reset git config.
```

### Version

**GitHub Desktop version:** 1.0.3
**OS version:** Win10 (10.0.15063)

### Steps to Reproduce

Not sure how to repro, but it happens each time I start the app.

**Expected behavior:** No error on startup

**Actual behavior:** Error on startup

**Reproduces how often:** Each time I start the app

### Logs

The first this shows in my logs is on 9/25

```
2017-09-25T04:16:57.446Z - info: [ui] Stats reported.
2017-09-25T04:17:16.490Z - info: [ui] Executing fetch: git -c credential.helper= fetch --progress --prune origin (took 4.077s)
2017-09-25T04:27:51.897Z - info: [ui] Executing fetch: git -c credential.helper= fetch --progress --prune origin (took 1.790s)
2017-09-25T04:32:53.095Z - info: [ui] Executing fetch: git -c credential.helper= fetch --progress --prune origin (took 2.048s)
2017-09-25T04:35:58.026Z - info: [ui] launching: 1.0.1 (Windows 10.0.15063)
2017-09-25T04:35:58.299Z - error: [ui] `git lfs install --skip-repo` exited with an unexpected code: 2.
Error running git [config --global filter.lfs.clean git-lfs clean -- %f]: 'warning: filter.lfs.clean has multiple values
error: cannot overwrite multiple values with a single value
       Use a regexp, --add or --replace-all to change filter.lfs.clean.' 'exit status 5'
Run `git lfs install --force` to reset git config.

2017-09-25T04:42:42.092Z - info: [ui] Executing pull: git -c credential.helper= pull --no-rebase --progress origin (took 4.599s)
2017-09-25T04:43:25.977Z - info: [ui] Executing clone: git -c credential.helper= lfs clone --recursive --progress --skip-repo --progress -- https://github.com/equineexchange/EquineExchange.ImageHosting.git C:\dev\equineexchange\EquineExchange.ImageHosting (took 1.923s)
```

```
2017-10-08T06:36:15.850Z - info: [ui] launching: 1.0.3 (Windows 10.0.15063)
2017-10-08T06:36:16.018Z - error: [ui] `git lfs install --skip-repo` exited with an unexpected code: 2.
Error running git [config --global filter.lfs.clean git-lfs clean -- %f]: 'warning: filter.lfs.clean has multiple values
error: cannot overwrite multiple values with a single value
       Use a regexp, --add or --replace-all to change filter.lfs.clean.' 'exit status 5'
Run `git lfs install --force` to reset git config.

2017-10-08T06:36:16.262Z - info: [ui] Executing getStatus: git status --untracked-files=all --branch --porcelain=2 -z (took 1.048s)
2017-10-08T06:36:16.571Z - info: [ui] Stats reported.
2017-10-08T06:36:31.498Z - info: [ui] Executing fetch: git -c credential.helper= fetch --progress --prune origin (took 1.539s)
```

**Comment by shiftkey** [comment]
@xt0rted this is related to multiple values being found in your global `~/.gitconfig`, and Git freaking out when it needs to update this.

We fixed this in #2951 for a slightly different issue, but for a workaround you can cleanup the duplicate entries in `~/.gitconfig` which will make this go away.

**Comment by shiftkey** [comment]
@technoweenie aside from handling this error in the app, do you think it's fine to re-run this with a `--force` to overwrite? Or are there some other cases we need to be cautious about?

**Comment by shiftkey** [comment]
Looks like we need to fix this in Git LFS itself. I've opened a PR over there to talk about this further https://github.com/git-lfs/git-lfs/pull/2659

**Comment by shiftkey** [comment]
That PR has been merged, and when `v2.3.4` lands I'll update our Git tooling to get this fix and close this out.
